### PR TITLE
Updated configs to add handler pattern.

### PR DESCRIPTION
**Pull request by parkeradam**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -197,4 +197,397 @@ versioning {
...
+        "read:individuals-income-hmcts-c3" {
+            fields = ["H", "J", "K", "M", "N", "O", "R", "T", "U", "AI", "AP", "BC", "BD", "BE", "BF", "BG"]
+        }
+
+        "read:individuals-income-hmcts-c4" {
+            fields = ["A", "B", "C", "D", "E", "F", "G", "H", "J", "AJ", "AO", "AP", "BC", "BD", "BE", "BF"]
+        }
+
+        "read:individuals-income-lsani-c1" {
+            fields = ["I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "AA", "AB", "AF", "AH", "AI", "AL", "AM", "AN", "AP", "AQ", "BQ", "BR", "BS", "BT", "BU", "BV", "BW", "BX", "AR", "AS", "AT", "AU", "AV", "AW", "AX", "AY", "AZ", "BA", "BB", "BC", "BD", "BE", "BF", "BG", "BH", "BI", "BJ", "BK", "BL", "BM", "BN", "BO", "BP"
```
Comment (ajchilts) [inline conf/application.conf:414]: Why separate the fields onto newlines at the top of the file and then go to one line here? If length > 80 chars then drop to newlines? Maybe 10 per line? Just play about with the best option for maintainability.

File: conf/application.conf [hunk 2]
```diff
@@ -197,4 +197,397 @@ versioning {
...
+        "read:individuals-income-lsani-c3" {
+            fields = ["I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "AA", "AB", "AF", "AH", "AI", "AL", "AM", "AN", "AP", "AQ", "BQ", "BR", "BS", "BT", "BU", "BV", "BW", "BX", "AR", "AS", "AT", "AU", "AV", "AW", "AX", "AY", "AZ", "BA", "BB", "BC", "BD", "BE", "BF", "BG", "BH", "BI", "BJ", "BK", "BL", "BM", "BN", "BO", "BP"
+        }
+
+        "read:individuals-employments-nictsejo-c4" {
+            fields = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "N", "O", "P", "Q", "R", "S", "T", "V", "W", "X", "AJ", "AN", "AP", "AQ", "BQ", "BR", "BS", "BT", "BU", "BV", "BW", "BX", "AR", "AS", "AT", "AU", "AV", "AW", "AX", "AY", "BC", "BD", "BE", "BF", "BH", "BI", "BJ", "BK", "BL", "BM", "BN"]
+        }
+    }
+
+    endpoints {
```
Comment (ajchilts) [inline conf/application.conf:426]: Also this endpoint config looks different to that in the test config used bu the code for testing the handler pattern. 

```
    endpoints = List(
      EndpointConfig(
        name = mockEndpoint1,
        link = "/a/b/c?matchId=<matchId>{&fromDate,toDate}",
        fields = Map(
          "A" -> "payments",
          "B" -> "employer/employerName=\"foo bar\"",
          "C" -> "employer/employerAddress/line1",
          "D" -> "employer/employerAddress/line2",
          "E" -> "employer/employerAddress/line3",
          "F" -> "employer/employerDistrictNumber",
          "G" -> "employer/employerSchemeReference",
          "H" -> "employmentStartDate",
          "I" -> "employmentLeavingDate",
          "J" -> "employmentPayFrequency",
          "K" -> "employeeAddress",
          "L" -> "payrollId"
        )
      ),
```

I think we will need to align to this name, link, fields 

Name is the string name of the endpoint used by the endpoint to fetch the scopes. 

Link is the full url for HAL 

Fields are the fields. 

@owen-opencast  can help confirm this?

**Review by ajchilts — CHANGES_REQUESTED** [review]

File: conf/application.conf [hunk 3]
```diff
@@ -197,4 +197,397 @@ versioning {
...
+        incomeSaSource {
+            endpoint = "/individuals/income/sa/source"
```
Comment (ajchilts) [inline conf/application.conf:428]: Update to correct HAL link and ensure endpoint keys match those in the controllers `getEndPointScopes` invocation.

**Review by ajchilts — CHANGES_REQUESTED** [review]

File: conf/application.conf (hunk 3 above)
Comment (parkeradam) [reply conf/application.conf:428]: The controllers dont currently have this as this is purley the config. We have not added the handler pattern into this repo as of the time of this ticket. These keys can be updated in controllers/config when this gets added in.

Comment (parkeradam) [reply conf/application.conf:428]: I have updated the HAL Links however

File: conf/application.conf (hunk 1 above)
Comment (parkeradam) [reply conf/application.conf:414]: Seperated into chunks of 10. Reads a lot better, good shout.

File: conf/application.conf [hunk 4]
```diff
@@ -197,4 +197,268 @@ versioning {
...
+                BM = "paye/employeePensionContribs/paid"
+                BN = "paye/employeePensionContribs/notPaid"
+                BO = "paye/dednsFromNetPay"
+                BP = "paye/statutoryPayYTD/parentalBereavement"
+                BQ = "paye/totalEmployerNICs/InPayPeriod1"
+                BR = "paye/totalEmployerNICs/InPayPeriod2"
+                BS = "paye/totalEmployerNICs/InPayPeriod3"
+                BT = "paye/totalEmployerNICs/InPayPeriod4"
+                BU = "paye/totalEmployerNICs/ytd1"
+                B = "paye/totalEmployerNICs/ytd2"
```
Comment (owen-opencast) [inline conf/application.conf:458]: Should this be BV?

**Review by owen-opencast — CHANGES_REQUESTED** [review]

**Review by owen-opencast — APPROVED** [review]

**Review by ajchilts — APPROVED** [review]

**Pull request merged**